src/addons/Textarea/Textarea.js

```diff
@@ -1,40 +1,19 @@
 import React, {Component, PropTypes} from 'react';
-import Field from 'src/collections/Form/Field';
+import classNames from 'classnames';
 
-class Textarea extends Component {
+export default class Textarea extends Component {
   static propTypes = {
-    dataContent: PropTypes.string,
+    className: PropTypes.string,
     label: PropTypes.string,
-    name: PropTypes.string,
-    onChange: PropTypes.func,
-    ref: PropTypes.string,
-    rows: PropTypes.oneOfType([
-      PropTypes.string,
-      PropTypes.number
-    ]),
-    value: PropTypes.string,
-  };
-
-  static defaultProps = {
-    rows: '3',
-    ref: 'textarea',
-  };
-
-  handleChange = (e) => {
-    this.props.onChange(e);
   };
 
   render() {
+    let classes = classNames(
+      'sd-textarea',
+      this.props.className
+    );
     return (
-      <Field label={this.props.label}>
-        <textarea
-          {...this.props}
-          onChange={this.handleChange}
-          data-content={this.props.dataContent}
-        />
-      </Field>
```

Comment:
We do not wrap our components with extra components.  This dictates usage and limits reusability.  We instead go for single purpose generic components that are more reusable.